CSC8503/TutorialGame.h

```diff
@@ -83,10 +86,22 @@ namespace NCL {
 			Mesh*	capsuleMesh = nullptr;
 			Mesh*	cubeMesh	= nullptr;
 			Mesh*	sphereMesh	= nullptr;
-
+			
 			Texture*	basicTex	= nullptr;
 			Shader*		basicShader = nullptr;
 
+
+			//Animation Thing
+			Mesh* soldierMesh = nullptr;
+			MeshAnimation* soldierAnimation = nullptr;
+			MeshMaterial* soldierMaterial = nullptr;
+			Shader* soldierShader = nullptr;
+			vector<GLuint> matTextures;
+			AnimationSystem* animation;
+
+			int currentFrame;
+			float frameTime;
```

Comment:
Also are we sure we want this in tutorial game for animation stuff? If we're going to have multiple animated things surely this would work better elsewhere?

Reply:
It seems I forgot to delete it, And I tried different places to run these two variables `mCurrentFrame`

 and `mFrameTime`, finally decided to define them in AnimationObject.